src/sections/messages/styles.css

```diff
@@ -0,0 +1,194 @@
+.react-rainbow-admin-messages{
+    display: flex;
+    min-height: calc(100vh - 68px);
+}
+
+.react-rainbow-admin-messages_contacts-container{
+    width: 30%;
+    background-color: #f4f6f9;
+}
+
+.react-rainbow-admin-messages_contacts-search {
```

Comment:
let's be consistent put an space before the bracket in all css block like here:
good:
```.react-rainbow-admin-messages_contacts-search {}```
bad:
```.react-rainbow-admin-messages_contacts-search{}```